services-js/registry-certs/client/death/checkout/PaymentContent.tsx

```diff
@@ -34,31 +40,56 @@ interface Props {
 }
 
 interface State {
-  touchedFields: Partial<{ [key in keyof OrderInfo]: boolean }>;
   tokenizationError: null | string;
 }
 
+export interface BillingInfo {
+  storeBilling: boolean;
+
+  cardholderName: string;
+  cardLast4: string;
+
+  billingAddressSameAsShippingAddress: boolean | string; // formik needs this value as a string
```

Comment:
I'd like to talk through this. We should tighten up the types to ensure that you can’t accidentally change `Order`’s `billingAddressSameAsShippingAddress` to a string.

Reply:
I’d love your insight on how best to handle this, because I don’t particularly like this solution. The issue is that `input type="radio"`’s value is a string, so Formik expects the same. The `BillingInfo` interface is used in several places, and I didn’t think it made sense to change the value to a string overall (it makes much more sense as a boolean). I’m not sure that it makes more sense to create a new interface specifically to be used for the form values within Formik?